modules/adding-comments.adoc

```diff
@@ -1,15 +1,22 @@
 // Module included in the following assemblies:
 //
 // * operating/manage-vulnerabilities.adoc
+// * operating/evaluate-security-risks.adoc
+//
+// Set the values for the following, before using this module:
+//:smallporv: process
+//:smallpluralporv: processes
+//:capsporv: Process
+//:capspluralporv: Processes
 :_module-type: PROCEDURE
 [id="adding-comments_{context}"]
 = Adding comments
 
-Comments allow you to add text notes to violations, so that everyone in the team can check what is happening with a violation.
+Comments allow you to add text notes to {smallpluralporv}, so that everyone in the team can check what is happening with a {smallporv}.
 
 .Prerequisites
 
-* To add and remove comments, you need a role with `write` permission for the resource you are modifying. For example, to add comments on violations, your role must have `write` permission for the `Alert` resource.
+* To add and remove comments, you need a role with `write` permission for the resource you are modifying. For example, to add comments on {smallpluralporv}, your role must have `write` permission for the `Alert` resource.
```

Comment:
@bergerhoffer Heads up. I've configured some variables in the modules. Because the functionality is exactly is the same.

Let me know if I should duplicate the modules instead. However, I prefer it this way.

Reply:
I've also asked if this is allowed in the FCC PV2 adopters channel.

Reply:
And I've got a confirmation. We can use this and its recommended that we add a default value for this variable in the common attributes file.

Reply:
@bobfuru FYI

Reply:
I'm open to this, especially if it is allowed by PV2, but I'll defer to @bergerhoffer. I seem to recall a recent team discussion about limiting our commonattributes file to more generalized variables such as product names and such.

Reply:
The proposal to add variables in the common attributes file was from someone else and not @bergerhoffer.

It was from the question I posted in the PV2 adopters channel.

> Question: I have a module which appears in multiple assemblies but depending upon the assembly it lists different option names. Can I do the following:
> module_1: . In the *Enable Notifications* dialog box, select the check box for the {toolname} notifier.
> assembly_1::toolname: ABC
> include: modules/module_1.adoc[leveloffset=+1]
> assembly_2::toolname: XYZ
> include: modules/module_1.adoc[leveloffset=+1]
> Or are there some guidelines already for such cases?
> XXXXXXXXXXXX, Jun 28, 12:53 PM
> From a technical perspective, yes - that is possible
> There are no restrictions against specifying local attributes in assemblies, and the proposed structure will build correctly
> From a practical perspective, because it must also be possible to build modules independently of an assembly, I'd recommend specifying a default value for that attribute in your attributes file
> We still need to implement support for building the same module in more than one way, but that's where we are today

I can remove those variables from the common attribute files if that's required.

Reply:
Thanks for the heads up (sorry just seeing this thread now). I don't see a problem with doing this - we do something similar with a few files in the main branch, so it shouldn't be an issue.

The FYI at the top of the module to set those attributes is good. The only thing I'd question is whether or not to set a default value in the common attributes file. Not because I think we need to limit what goes into it (like what @bobfuru mentioned the core team decided to do), but because that default value might be wrong and someone might not notice. 

For example, our use of it for the [technology preview file](https://raw.githubusercontent.com/openshift/openshift-docs/master/modules/technology-preview.adoc) specifically unsets the `:FeatureName:` attribute so that mistakes are easily spotted when viewing the build.

So anyways... just something to think about. But it's fine to keep the default value if you'd like.

Reply:
I'm keeping it as it is as recommended by a PV2 team member.